code/script.js

```diff
@@ -229,97 +232,162 @@ const setSecret = () => {
 
 // This function to start (and restart) the game
 const start = () => {
+  document.getElementById('winOrLose').style.display = 'none'
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
   // What else should happen when we start the game?
+generateBoard ()
+setSecret ()
+selectQuestion ()
```

Comment:
Nice and easy to read section. I like that you kept the comments for clarity as well!